src/view/MainViewManager.js

```diff
@@ -845,6 +846,35 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Switch between panes
+     */
+    function switchPaneFocus() {
+        var $firstPane = $('#first-pane'), $secondPane = $('#second-pane');
+        if($firstPane.hasClass('active-pane')) {
+            $secondPane.click();
+        }
+        else {
+            $firstPane.click();
+        }
+    }
+
+    /**
+     * Unit test for switching from first pane to second pane
```

Comment:
@arthur801031 Dang, I remembered that SimulateKeyEvent has modifier keys support but it seems not: https://github.com/adobe/brackets/blob/master/test/spec/SpecRunnerUtils.js#L998

Let's modify the tests so that you use `CommandManager.executeCommand` to run `cmd.switchPaneFocus` and check that panes have switches after running that command.

Sorry for the extra trouble. 👍